<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="live_widgets::ToolButton" name="inputType">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>41</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">    border-top-left-radius: 4px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../creator-live/icons/icons.qrc">
          <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">* {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #3c3c3c, stop: 1 #262627);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="live_widgets::RotatedLabel" name="inputName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>handelgotdlig</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>No Input</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="live_widgets::ToolButton" name="inputPaint">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../creator-live/icons/icons.qrc">
          <normaloff>:/icons/paint.png</normaloff>:/icons/paint.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="live_widgets::ToolButton" name="inputNew">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>handelgotdlig</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string>R</string>
        </property>
        <property name="icon">
         <iconset resource="../creator-live/icons/icons.qrc">
          <normaloff>:/icons/addChain.png</normaloff>:/icons/addChain.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="missile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>52</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../creator-live/icons/icons.qrc">
          <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
    background: #4d4c4d;
    font-family: handelgotdlig;
    font-weight: 200;
}



 QComboBox {
     border: 1px solid #141414;
     border-radius: 4px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
     color: #e8e8e8;
 }

 QComboBox:editable {
    background: #5b5a5b
 }

 QComboBox:!editable, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
      stop: 0 #3c3c3c, stop: 1 #262627);
 }

 /* QComboBox gets the &quot;on&quot; state when the popup is open */
 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
     background: #5b5a5b
 }

 QComboBox:on { /* shift the text when the popup opens */
     padding-top: 3px;
     padding-left: 4px;
 }

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }

QComboBox QAbstractItemView {
    border: 1px solid #141414;
    selection-background-color: #5b5a5b;
}

 QSpinBox {
     /*padding-right: 15px; make room for the arrows */
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
    color: #e8e8e8;
     border: none;
     border-radius: 4px;
     /*border-width: 3;*/
}

QSpinBox::up-button {
    border: none;
}

QSpinBox::down-button {
    border: none;
}

QLineEdit {
    border: 1px solid #141414;
    border-radius: 4px;
    color: #e8e8e8;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #3c3c3c, stop: 1 #262627);
    selection-background-color: #e8e8e8;
    selection-color: #262627
}

QProgressBar {
    border: 1px solid #141414;
    border-radius: 4px;
    text-align: center;
 }


 QProgressBar::chunk {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
    width: 20px;
 }

QFrame, QLabel, QToolTip {
   color: #e5e5e5;
   /*border: 1px solid #141414;*/
}

QFrame[style=&quot;plain&quot;] {
    border: 100px solid pink;
}

QAbstractButton {
   padding: 3px;
   border: 1px solid #141414;
   border-radius: 4px;
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #3c3c3c, stop: 1 #262627);
    color: #e8e8e8;
}
QAbstractButton::hover {
   background-color: #5b5a5b;
}

QAbstractButton::disabled {
    color: #262627;
}

QAbstractButton::checked {
    background-color: #1c1c1c;
}

 QTabWidget::pane { /* The tab widget frame */
     border-top: 1px solid #141414;
     position: absolute;
     top: -0.5em;
 }

 QTabWidget::tab-bar {
     alignment: center;
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #3c3c3c, stop: 1 #262627);

    border: 1px solid #141414;

     /*border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 2px;
     color: #e8e8e8;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: #5b5a5b;
 }

 QTabBar::tab:selected {
     border-color: #5b5a5b;
     color: #141414;
     /*border-bottom-color: #C2C7CB; same as pane color */
 }

 QSlider::groove:horizontal {
 border: 1px solid #141414;
     height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #3c3c3c, stop: 1 #262627);
     margin: 2px 0;
 }

 QSlider::handle:horizontal {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #3c3c3c, stop: 1 #262627);     border: 1px solid #5c5c5c;
     width: 18px;
     /*margin: -2px 0;  handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 3px;
 }

  QScrollBar:vertical {
      border: 0px solid transparent;
      background: transparent;
      width: 4px;
      margin: 0px 0 0px 0;
  }
  QScrollBar::handle:vertical {
      background: #e8e8e8;
      min-width: 20px;
      border: 2px solid #e8e8e8;
      border-radius: 2px;
  }
  QScrollBar::add-line:vertical {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }

  QScrollBar::sub-line:vertical {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }
  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
      border: 0px solid grey;
      width: 0px;
      height: 0px;
      background: white;
  }

  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }



  QScrollBar:horizontal {
      border: 0px solid transparent;
      background: transparent;
      height: 4px;
      margin: 0px 0 0px 0;
  }
  QScrollBar::handle:horizontal{
      background: #e8e8e8;
      min-height: 20px;
      border: 2px solid #e8e8e8;
      border-radius: 2px;
  }
  QScrollBar::add-line:horizontal {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }

  QScrollBar::sub-line:horizontal {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }
  QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
      border: 0px solid grey;
      width: 0px;
      height: 0px;
      background: white;
  }

  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }





QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 20px;
border-radius: 4px;
}

QSlider::groove:vertical {
border: 1px solid #bbb;
background: white;
width: 50px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #444, stop: 1 #bbb);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbb, stop: 1 #555);
border: 0px solid #777;
height: 40px;
border-radius: 4px;
border-top-left-radius: 4px;
border-top-right-radius: 0px;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 0px;
}

QSlider::sub-page:vertical {
background: qlineargradient(x1: 0, y1: 1,    x2: 0, y2: 0,
    stop: 0 #bbb, stop: 1 #444);
background: qlineargradient(x1: 0, y1: 1, x2: 1, y2: 0.2,
    stop: 0 #555, stop: 1 #bbb);
border: 0px solid #777;
width: 53px;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
}

QSlider::add-page:horizontal {
background: qlineargradient(x1: 0, y1: 1,    x2: 0, y2: 0,
    stop: 0 #bbb, stop: 1 #444);
background: qlineargradient(x1: 0, y1: 1, x2: 1, y2: 0.2,
    stop: 0 #555, stop: 1 #bbb);
border: 0px solid #777;
height: 40px;
border-top-left-radius: 0px;
border-top-right-radius: 4px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 4px;
}

QSlider::add-page:vertical {
background: qlineargradient(x1: 0, y1: 1,    x2: 0, y2: 0,
    stop: 0 #444, stop: 1 #bbb);
background: qlineargradient(x1: 1, y1: 1, x2: 0, y2: 0.2,
    stop: 0 #bbb, stop: 1 #555);
border: 0px solid #777;
width: 53px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 4px;
}

QSlider::handle:horizontal {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
border: 1px solid #777;
width: 53px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 10px;
margin: 2px;
}

QSlider::handle:vertical {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
border: 1px solid #777;
height: 33px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 10px;
margin: 2px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::sub-page:vertical:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::add-page:vertical:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

QSlider::handle:vertical:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

QFrame {
background: #1c1c1c;
border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 4px;
border-bottom-right-radius: 4px;
}

QListWidget {
background: #1c1c1c;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>345</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>345</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;Choose an Input. &lt;/span&gt;&lt;span style=&quot; font-size:12pt; color:#adadad;&quot;&gt;See Live &amp;gt; &amp;quot;Audio Setup&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="input_combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>345</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>345</width>
          <height>350</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>handelgotdlig</family>
          <weight>25</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">* {
spacing: 0px;
color: black;
}

QListWidget {
   alignment: center;
   spacing: 23;
}

QListWidget::item {
   height: 36px;
   border: 1px solid #141414;
   border-radius: 4px;
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #3c3c3c, stop: 1 #262627);
    color: #e8e8e8;
}

QListWidget::item::selected {
background-color: #5b5a5b
}


QListWidget::item::hover {
   background-color: #5b5a5b;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Fixed</enum>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Loading...</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>live_widgets::ToolButton</class>
   <extends>QToolButton</extends>
   <header>live_widgets/toolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>live_widgets::RotatedLabel</class>
   <extends>QLabel</extends>
   <header location="global">live_widgets/rotatedlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../creator-live/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
